impeller/renderer/backend/vulkan/capabilities_vk.h

```diff
@@ -143,6 +116,46 @@ enum class OptionalDeviceExtensionVK : uint32_t {
   kLast,
 };
 
+//------------------------------------------------------------------------------
+/// @brief      A device extensions that may be available on Android platforms.
+///             Without the presence of these extensions on Android, certain
+///             features cannot be used, like AHB swapchains.
+///
+///             Platform agnostic code can still check if these Android
+///             extensions are present.
+///
+enum class OptionalAndroidDeviceExtensionVK : uint32_t {
```

Comment:
There is nothing Android specific about these extensions, you can move them to `OptionalDeviceExtensionVK`.